.gitignore

```
@@ -75,3 +75,6 @@ move_compile/
 
 # binaries
 sui-binaries/
+
+# move test folder
+move-test/
```

```suggestion
test/move
```
keep it inside the test folder for clarity?